src/assignment.js

```diff
@@ -1,14 +1,17 @@
-// do not edit this
-let firstNumber = 10
-firstNumber = 0
+// Do not edit this
+let firstNumber = 20;
+firstNumber = 0;
 
 // TODO: 1. Set the value of firstNumber below so the tests pass
+firstNumber = 20;
 
 // TODO: 2. Change the code below so that the tests pass
-const secondNumber = 0 // edit this value
+const secondNumber = 0; // edit this value
+// Change the value of secondNumber here
+const secondNumber = 42;
```

Comment:
@loza01 This causes a syntax error, you can't redeclare a `const`

Reply:
I removed that now but still there is syntaxError that shows 'firstNameExport' has already been declared. how should I fix that?

Reply:
I got it
but LBNL still one is still failing giving me an error of Variable Declaration: age is 35
  Message:
    Expected 10 to equal 35.
  Stack:
    Error: Expected 10 to equal 35. and I'm still confused

Reply:
It's fixed now!

Reply:
Awesome :)